string-manipulation-1/src/get-first-char.js

```diff
@@ -0,0 +1,12 @@
+/* exported getFirstChar */
+// storage for the first character in a string.
+// for each character in a string, one by one,
+// get the first character value of the string and store it.
+// give back the location of stored first character of the string.
+function getFirstChar(string) {
+  var firstChar = [];
+  for (var i = 0; i < string.length; i++) {
+    firstChar = string[0];
+  }
```

Comment:
Notice that you are not making use of the `i` variable in any way, that is an indicator that this loop may not be necessary. You are just accessing one value, you can do that without a loop.